packages/workbox-build/src/entry-points/options/defaults.js

```diff
@@ -7,18 +7,22 @@
 */
 
 module.exports = {
+  babelPresetEnvTargets: ['chrome >= 56'],
   cleanupOutdatedCaches: false,
   clientsClaim: false,
   globFollow: true,
   globIgnores: ['**/node_modules/**/*'],
   globPatterns: ['**/*.{js,css,html}'],
   globStrict: true,
   importWorkboxFrom: 'cdn',
```

Comment:
Does this need to be changed too (now that we're generating a user-specific bundle)?

Reply:
We need some sort of default for the custom bundles, and `['chrome >= 56']` seemed like the one most in keeping with our overall compatibility goals.

At least folks have straightforward control over this now via configuring an alternative, which wasn't the case with our pre-built bundles.

Reply:
Ahh, I wasn't clear. This comment is referring to the `importWorkboxFrom` field.

Reply:
Ah, good catch. `importWorkboxFrom` was removed as a supported option, so I can remove it from the default list here.